define/types.go

```diff
@@ -100,6 +100,8 @@ type Secret struct {
 
 // BuildOutputOptions contains the the outcome of parsing the value of a build --output flag
 type BuildOutputOption struct {
+	Type     string
```

Comment:
What are the values of `Type`? Shouldn’t this be an enum?

Reply:
(At the very least, if this can’t be done with native Go features, the values should be documented.)

Reply:
Thanks for pointing this out, updated it to use an enum here. Should be extensible and readable.